.github/workflows/otel-fork-replace.yml

```diff
@@ -38,38 +38,39 @@ jobs:
-          go mod tidy
-          go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver=github.com/amazon-contributing/opentelemetry-collector-contrib/receiver/prometheusreceiver@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus=github.com/amazon-contributing/opentelemetry-collector-contrib/pkg/translator/prometheus@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs=github.com/amazon-contributing/opentelemetry-collector-contrib/internal/aws/cwlogs@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter=github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver=github.com/amazon-contributing/opentelemetry-collector-contrib/receiver/awsxrayreceiver@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/amazon-contributing/opentelemetry-collector-contrib/override/aws=github.com/amazon-contributing/opentelemetry-collector-contrib/override/aws@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
-          go mod edit -replace github.com/amazon-contributing/opentelemetry-collector-contrib/pkg/stanza=github.com/amazon-contributing/opentelemetry-collector-contrib/pkg/stanza@${{ steps.get-latest-commit.outputs.sha }}
-          go mod tidy
+          packages=(
+            exporter/awscloudwatchlogsexporter
+            exporter/awsemfexporter
+            exporter/awsxrayexporter
+            extension/awsmiddleware
+            extension/awsproxy
+            internal/aws/awsutil
+            internal/aws/containerinsight
+            internal/aws/cwlogs
+            internal/aws/k8s
+            internal/aws/proxy
+            internal/aws/xray
+            internal/kubelet
+            pkg/translator/prometheus
+            processor/resourcedetectionprocessor
+            receiver/awscontainerinsightreceiver
+            receiver/awsxrayreceiver
+            receiver/prometheusreceiver
+          )
+          for p in ${packages[@]}; do
+            go mod edit -replace github.com/open-telemetry/opentelemetry-collector-contrib/$p=github.com/amazon-contributing/opentelemetry-collector-contrib/$p@${{ steps.get-latest-commit.outputs.sha }}
+            go mod tidy
+          done
+          
+          amazonContributingPackages=(
+            override/aws
+            pkg/stanza  
```

Comment:
This should be in the previous group (`packages`) since we're replacing the upstream contrib package.

Reply:
Done.